research-hub-web/src/app/components/org-unit-details/org-unit-details.component.ts

```diff
@@ -6,8 +6,12 @@ import { Location } from '@angular/common';
 import { AnalyticsService } from '@services/analytics.service';
 import { ListItem } from '@model/ListItem';
 import { AppComponentService } from '@app/app.component.service';
-import { RoleTypeId } from '@services/options.service';
 
+enum RoleTypeId {
```

Comment:
This one we're less likely to reuse in different components, but still wouldn't hurt to put somewhere that it could be consumed by other components if we need it